html/index.html

```diff
@@ -12,38 +12,35 @@
 
 
 var width = 960,
-    height = 1160;
+    height = 600;
 
 var svg = d3.select("body").append("svg")
     .attr("width", width)
     .attr("height", height);
 
-var path = d3.geo.path()
-    // .projection(projection);
+var projection = d3.geo.albers()
```

Comment:
Ah, it's largely trial and error, combined with some experience of doing this before.

Works for me. Thanks again for the pointers.